README.md

```diff
@@ -1,3 +1,3 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+To set the environment variables, you will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=, with the correct database name for that environment (see /db/setup.sql for the database names). Double check that these .env files are .gitignored.
```

Comment:
You dont need to have the final sentence. This is something only you need to do, so everyone who clones this after you should have your updated .gitignored file.